libraries/botbuilder-lg/src/templatesParser.ts

```diff
@@ -175,6 +175,22 @@ export class TemplatesParser {
         return templates;
     }
 
+    public static antlrParseTemplates(text: string, source: string): FileContext {
+        if (!text || text.trim() === '') {
```

Comment:
Missing documentation of public element.

Reply:
#Solved